### PR TITLE
add try/catch 

### PR DESCRIPTION
**Pull request by dothebart**
so in case the server didn't output the expected structure we see a bit better error messages

### CODE REVIEW

File: js/common/modules/@arangodb/aql/explainer.js [hunk 1]
```diff
@@ -205,12 +212,21 @@ function printRules(rules) {
   stringBuilder.appendLine(section('Optimization rules applied:'));
   if (rules.length === 0) {
     stringBuilder.appendLine(' ' + value('none'));
-  } else {
+  } else try {
```
Comment (ObiWahn) [inline js/common/modules/@arangodb/aql/explainer.js]: Not sure if `} else try {` is valid.

**Comment by ObiWahn** [comment]
Please add a Jenkins run!

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8910/

File: js/common/modules/@arangodb/aql/explainer.js (hunk 1 above)
Comment (dothebart) [reply js/common/modules/@arangodb/aql/explainer.js]: it is valid.

**Pull request closed without merging**